RemoveGtaProjects.ps1

```diff
@@ -0,0 +1,24 @@
+$gta_guids = @(
```

Comment:
I'm thinking this script may go to Tools since it is not used in normal dev workflow.